src/main/java/de/tum/in/www1/artemis/web/rest/dto/AtheneDTO.java

```diff
@@ -0,0 +1,34 @@
+package de.tum.in.www1.artemis.web.rest.dto;
+
+import java.util.*;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.TextBlockType;
+import de.tum.in.www1.artemis.domain.TextCluster;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class AtheneDTO {
+
+    public List<TextBlock> blocks = new ArrayList<>();
```

Comment:
please use private attributes and getters and setters here

Reply:
We are not consistent with how we handle this for DTOs in the code right now. I'd argue for dump DTOs that it is unneeded to add getters/setters.